### DISCUSSION TITLE
Add an "embed" button on the markdown toolbar

### DISCUSSION THREAD

**Discussion by JennieOcken** [forem/forem#16636] [Feature Requests]
Based on a conversation [here](https://forem.dev/foremteam/changelog-a-new-and-streamlined-way-to-embed-rich-content-4h58), we should including the rich content embed as a button inside the markdown toolbar.  This would help our new users find this option.

**Comment by ildi-x** [comment]
Anything that helps new users easily share content is a win for Forem! Thank you for making note of this, I hope to see it implemented in the future.

**Comment by aitchiss** [comment]
I think this would be a pretty quick win 😄

**Comment by JennieOcken** [comment]
Added as an [issue ](https://github.com/forem/forem/issues/16780) in our forem/forem queue.
